boot271/Makefile

```diff
@@ -16,5 +16,10 @@ LDFLAGS =
 PSP_LARGE_MEMORY = 1
 BUILD_PRX = 1
 
+EXTRA_CLEAN = clean_eboot

```

Comment:
This could also be rewritten to remove the `release` target at the top and then
having this here instead of the extra clean target:

```Makefile
EXTRA_TARGETS = EBOOT.PBP
PSP_EBOOT_TITLE = PSPLink v3.0 OE
PSP_EBOOT_ICON = psplink.png
```

Then `build_prx.mak` will take care of it.

Reply:
Initially I thought that as it would been inline with other makefiles. But I had a look at the `build_prx.mak` in my sdk and there are no references to `EXTRA_TARGETS`. Seems like the `BUILD_PRX` flag is only relevant to `build.mak` which includes `EXTRA_TARGETS`. It seems switching to `build.mak` would required a lot more changes which I think may be outside the scope of this pull request. 

I could be wrong though. Still only have a shallow knowledge of this SDK but willing to learn more.

Reply:
You are right. Makes more sense to have it this way then. Thanks!